changelog/0032-upgrade-elasticsearch.yml

```
@@ -0,0 +1,50 @@
+---
+title: ES upgrade from 1.7.6 to 2.4.6
+key: upgrade-elasticsearch
+date: 2020-02-05
+optional_per_env: yes
+update_steps: |
+  1.stop the site
+  commcare-cloud <env> downtime start
+    
+  2.Change the es version and checksum in commcare-cloud/environments/<env>/public.yml 
+  elasticsearch_version: 2.4.6
+  elasticsearch_download_sha256: 5f7e4bb792917bb7ffc2a5f612dfec87416d54563f795d6a70637befef4cfc6f.
+  ELASTICSEARCH_MAJOR_VERSION: 2
+    
+  3.update the local settings
```

Markdown requires a space between the `3.` and the text (same comment for all points)
```suggestion
  3. Update the local settings
```